controllers/server.go

```diff
@@ -18,16 +19,24 @@ func serverHandlers(r *mux.Router) {
 	// r.HandleFunc("/api/server/addnetwork/{network}", securityCheckServer(true, http.HandlerFunc(addNetwork))).Methods(http.MethodPost)
 	r.HandleFunc(
 		"/api/server/health",
-		http.HandlerFunc(func(resp http.ResponseWriter, req *http.Request) {
+		func(resp http.ResponseWriter, req *http.Request) {
 			resp.WriteHeader(http.StatusOK)
 			resp.Write([]byte("Server is up and running!!"))
-		}),
-	)
+		},
+	).Methods(http.MethodGet)
+	r.HandleFunc(
+		"/api/server/health",
+		func(w http.ResponseWriter, _ *http.Request) {
+			_ = syscall.Kill(syscall.Getpid(), syscall.SIGINT)
```

Comment:
can we add a log before the sigkill

Reply:
good idea

Reply:
https://github.com/gravitl/netmaker/pull/2672/commits/59add9dd04d2923ac7757467867c9d1e77fc6ca0